wordclock_interfaces/templates/scripts/scripts.js

```diff
@@ -0,0 +1,154 @@
+var ColorPicker = window.VueColorPicker;
+new Vue(
+{
+	el: '#app',
+	components: {
+                ColorPicker: ColorPicker
+	},
+	data: {
+		brightness: 50,
+                color: {
+                    hue: 50,
+                    saturation: 100,
+                    luminosity: 50,
+                    alpha: 1
+                },
+		apiData: undefined,
+		selectedPlugin: {
+		    name: 'time_default'
+		},
+		currentPlugin: {
+		    name: 'time_default'
+		},
+		switchWords: true,
+		switchMinutes: true,
+		switchBackground: false
+	},
+	methods: {
+		loadApi: function () {
+			this.$http.get('/api/plugin').
+			  then(this.successCallback2, this.errorCallback);
+			this.$http.get('/api/plugins').
+			  then(this.successCallback, this.errorCallback);
+		},
+		successCallback: function(response) {
+			this.apiData = response.data;
+			console.log('successCallback this.apiData:' , this.apiData);
+		},
+		successCallback2: function(response) {
+			this.currentPlugin = response.data.plugin;
+			console.log('successCallback this.currentPlugin:' , this.currentPlugin);
+		},
+		errorCallback: function(response) {
+			console.log('errorCallback response:' , response);
+		},
+		selectionChanged: function(selectedPlugin) {
+			console.log('selectionChanged: ', selectedPlugin);
+			var xmlhttp = new XMLHttpRequest();
+			xmlhttp.open("POST", "/api/plugin");
+			xmlhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
+			xmlhttp.send(JSON.stringify({ "name": selectedPlugin.name}));
+		},
+		buttonClick: function(buttonClicked) {
+			console.log('buttonClicked', buttonClicked );
+			var xmlhttp = new XMLHttpRequest();
+			xmlhttp.open("POST", "/api/button");
+			xmlhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
+			xmlhttp.send(JSON.stringify({ button: buttonClicked}));
+		},
+		setColour: function(r,g,b) {
+			console.log('setColour:' , r,g,b);
+			if (this.switchWords) {
+				type = "words"
+			};
+			if (this.switchMinutes) {
+				type = "minutes"
+			};
+			if (this.switchWords && this.switchMinutes) {
+				type = "all"
+			};
+			if (this.switchBackground) {
+				type = "background"
+			};
+			var xmlhttp = new XMLHttpRequest();
+			xmlhttp.open("POST", "/api/color");
+			xmlhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
+			xmlhttp.send(JSON.stringify({ "blue": b, "green": g, "red": r , "type": type}));
+		},
+		updateBrightness: function(brightness) {
+			console.log('updateBrightness:' , brightness);
+			var xmlhttp = new XMLHttpRequest();
+			xmlhttp.open("POST", "/api/brightness");
+			xmlhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
+			xmlhttp.send(JSON.stringify({ "brightness": brightness }));
+		},
+		setColourWheel: function(wheel) {
+			var r,g,b,type;
+			[g,r,b] = hslToRgb(wheel.hue/360, wheel.saturation/100, wheel.luminosity/100);
```

Comment:
Adopt to [r,g,b]
... results from fix for https://github.com/bk1285/rpi_wordclock/issues/99

Reply:
This was actually a temporary fix so I can at least set the right colour through the colour wheel.
The rest of the plugins still have red and green swapped.